### PR TITLE
Changing event detail layout

### PR DESCRIPTION
**Pull request by gingerzoealex**
Alteration to how the event information is displayed.

| Before | After |
| ----- | ----- | 
| muxer_eventsbefore |muxer_eventsafter |

### CODE REVIEW

**Comment by apoclyps** [comment]
@gingerzoealex The latest updates are starting to look really nice; On mobile, they look like they will be a huge improvement. 

The the DateTime row; I think it still needs a little bit of work. The date and event happening in are too similar. I think we could play around with the font weights of each and instead of seperating by a `-`, we should keep the  `event happening in x time` inside brackets: e.g. `( even happening in 1 day )`

What do you think above those proposed changes?

Also, CI is failing on a few linting errors; you can identify what these are locally by running 

```
docker-compose -f docker-compose-stage.yml run client-test npm run lint
```

However if you want to automatically fix these, you can add the `-- --fix flag` to do this automatically using:

```
docker-compose -f docker-compose-stage.yml run client-test npm run lint -- --fix
```

File: services/client/src/components/event/content.js [hunk 1]
```diff
@@ -18,13 +18,15 @@ const Content = ({ name, start, url, category }) => (
...
+    </div>
     <div className="event-time">
       <EventTime startTime={start} />
-    </div>
-    <div className="event-details">
-      {category} &nbsp;
       <TimeToEvent startTime={start} />
     </div>
+
+
```
Comment (alistairjcbrown) [inline services/client/src/components/event/content.js]: Can we nuke this extra whitespace?

File: services/client/src/components/event/time-to-event.js [hunk 2]
```diff
@@ -13,11 +13,11 @@ const TimeToEvent = ({ startTime }) => {
         span {
           color: #989fa5;
           font-style: italic;
-          font-weight: 200;
+          font-weight: 500;
         }
       `}</style>
       <span className="humanize">
-        ( {duration > 0 ? "next" : "last"} event {duration.humanize(true)} )
+        - {duration > 0 ? "next" : "last"} event {duration.humanize(true)}
```
Comment (alistairjcbrown) [inline services/client/src/components/event/time-to-event.js]: You may need to add a `&nbsp;` here to provide the space character in front of the `-`, or you could also try making the `-` a string literal so React knows the keep the space in front:
```js
<span className="humanize">
  {' - '}{duration > 0 ? "next" : "last"} event {duration.humanize(true)}
</span>
```

**Comment by apoclyps** [comment]
I'm going to close our this PR for the time being to avoid it rebuilding on Travis when changes are merged. I'll leave the branch behind so it can be reopened in the future.

**Pull request closed without merging**